yarn-project/aztec.js/src/wallet/base_wallet.ts

```diff
@@ -129,8 +129,8 @@ export abstract class BaseWallet implements Wallet {
   getPublicStorageAt(contract: AztecAddress, storageSlot: Fr): Promise<any> {
     return this.pxe.getPublicStorageAt(contract, storageSlot);
   }
-  addNote(note: ExtendedNote): Promise<void> {
-    return this.pxe.addNote(note);
+  addNote(note: ExtendedNote, account: AztecAddress): Promise<void> {
+    return this.pxe.addNote(note, account);
```

Comment:
This is weird and a prime example on why we should expose PXE directly: add a note to this siloed storage shouldn't be the wallet's responsability